src/components/App/index.tsx

```diff
@@ -1,14 +1,22 @@
 import React from "react"
 import logo from "../../assets/logo.svg"
+import classnames from "classnames"
 import "./index.css"
 
-type Props = Readonly<{ children?: React.ReactNode }>
+type Props = Readonly<{
+  children?: React.ReactNode
+  isAnimated?: boolean
```

Comment:
I might suggest that we change this to `notAnimated?: boolean` so that we don't need default props (of course I would also change the classname to `App-logo--notAnimated` to correspond). The reason I'm nitpicking is this will serve as the example for how to do this. I want to make sure we get it right now 😉